spec/spec.md

```diff
@@ -1713,15 +1713,17 @@ Schema: [`Demographics.schema.json`](Demographics.schema.json)
     [{ 
         "id" : "as-fert",
         "name": "Age-specific fertility",
-        "sources" : ["unwpp2015", "unwpp2017"],
-        "countries" : ["AFG"],
+        "source" : "unwpp2015",
+        "sources": ["unwpp2015"],
+        "countries": [],
```

Comment:
Idea is to remove `countries` and `sources` entirely. Leaving them in for now so that this is a non-breaking change. Once the portals have been updated to remove all references to these can remove from model: https://vimc.myjetbrains.com/youtrack/issue/VIMC-808

Reply:
Could we add this comment to the spec itself, so our intention isn't only preserved in this PR?